### PR TITLE
ci: release also to github

### PR DESCRIPTION
**Pull request by maiqueb**
**What this PR does / why we need it**:
The `make release` target will also perform a github release.

`make release` can be invoked:
  - manually
  - by the postsubmit hook executed once a PR is merged, and there's a version bump involved

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,8 +99,13 @@ prepare-minor:
 prepare-major:
 	./hack/prepare-release.sh major
 
+$(GITHUB_RELEASE): go.mod
+	go install ./vendor/github.com/aktau/github-release
+
 release: IMAGE_TAG = $(shell hack/version.sh)
 release: docker-build docker-push
+release: $(GITHUB_RELEASE)
+	TAG=$(IMAGE_TAG) GITHUB_RELEASE=$(GITHUB_RELEASE) DESCRIPTION=./version/description ./hack/release.sh
```
Comment (maiqueb) [inline Makefile]: The `description` part is not working ... Does anything scream bloody murder there, @phoracek  ?

Comment (maiqueb) [reply Makefile]: My bad!!! It says `empty token`, which I'm assuming is the missing github token.

**Comment by maiqueb** [comment]
Any clue on how we can test this, @phoracek ?

**Comment by phoracek** [comment]
@maiqueb let's just try it. Let's assume we don't have any users and it won't be disruptive.

**Comment by phoracek** [comment]
Try it = merge it and watch :popcorn:

**Pull request merged**